packages/plugin-interactive-tools/sources/commands/upgrade-interactive.tsx

```diff
@@ -102,13 +102,13 @@ export default class UpgradeInteractiveCommand extends BaseCommand {
         `red`, // major
         `yellow`, // minor
         `green`, // patch
-        `magenta`, // rc
+        `magenta`, // prerelease
       ];
 
       let color: string | null = null;
       let res = ``;
 
-      for (let t = 1; t < SEMVER_COLORS.length; ++t) {
```

Comment:
we start with `1`, but end with `length - 1`, so we missed the prerelease part of the semver.